tests/publiccloud/nvidia.pm

```diff
@@ -14,11 +14,11 @@ use utils;
 use publiccloud::utils;
 
 sub validate_nvidia {
-    validate_script_output("hwinfo --gfxcard", sub { /nVidia.*Tesla T4/mg });    # depends on terraform setup
-        # nvidia-smi is delivered by nvidia-compute-utils-G06. Not available on PublicCloud
-        # Check device files and loaded modules instead.
-    validate_script_output("lsmod", sub { m/nvidia/ }, fail_message => "nvidia module not loaded");
-    assert_script_run("ls -la /dev/{nvidia-modeset,nvidia-uvm-tools,nvidiactl,nvidia-uvm,nvidia0}", fail_message => "nvidia device files are missing");
+    # Check card name, depends on terraform setup
+    validate_script_output("hwinfo --gfxcard", sub { /nVidia.*Tesla T4/mg });
+    # Check loaded modules
+    validate_script_output("lsmod | grep nvidia", fail_message => "nvidia module not loaded");
```

Comment:
`validate_script_output` doesn't make sense with `grep`.  Use either `assert_script_run` or return to:

```suggestion
    validate_script_output("lsmod", sub { m/nvidia/ }, fail_message => "nvidia module not loaded");
```

Reply:
Ah yes I was thinking of assert_script_run. Sorry about that!